frontend/src/components/about/BetaAlertDiv.tsx

```diff
@@ -0,0 +1,58 @@
+import { useTranslation } from 'react-i18next';
+import Alert from '@mui/material/Alert';
+import CloseIcon from '@mui/icons-material/Close';
+import { Button } from '@mui/material';
+
+type BetaAlertDivProps = {
+  setAlertVisible: (isVisible: boolean) => void;
+};
+
+const BetaAlertDiv: React.FC<BetaAlertDivProps> = ({ setAlertVisible }) => {
```

Comment:
I’d recommend you to name the component simply ‘BetaAlert’. To me ‘div’ seems redundant.

Also, I think this alert does not need to be in a separate component. But I also want to hear what @RyotaMitaraiWeb thinks….?

Reply:
feels like an overkill to me, personally, it's not really abstracting or encapsulating anything that the component already doesn't.

Reply:
I moved it back into main component About.tsx.